<launch>
    <!-- Start the Oculus Reader Node -->
    <node name="oculus_reader" pkg="your_package_name" type="oculus_node.py" output="screen">
        <!-- Specify the interpreter (Python 3) -->
        <param name="use_python3" value="true"/>
    </node>

    <!-- TF2 setup, if needed for transforms -->
    <node name="tf2_ros" pkg="tf2_ros" type="static_transform_publisher" args="0 0 0 0 0 0 world oculus" />

    <!-- Publisher for OculusData messages -->
    <node name="oculus_publisher" pkg="rospy" type="publisher" output="screen">
        <param name="topic_name" value="/oculus_reader/data" />
        <param name="message_type" value="vr_control/OculusData" />
    </node>
</launch>
